### ISSUE TITLE
Add Shortcut file type, simplify shortcuts generally

### ISSUE THREAD

**Issue by inexorabletash** [a2stuff/a2d#751]
I was watching [a video about the Lisa](https://www.youtube.com/watch?v=pcYX-2uWwsk) and it put me in the mood to think about how the Macintosh was forced by limited time and lack of memory/ROM/disk resources to be extremely minimal. No bells and whistles. And like much great art, editing down often makes things better. Lots of the improvements to A2D have been by removing things to simplify the experience.

One thing that really stands out in A2D as surprising _complexity_ are the Shortcuts, called Selector items, in the original MouseDesk/A2D 1.1. From the perspective of "how can we make a simplified launcher, and turn desktop into an editor for the entries?" it makes sense. But it's a lot of added complexity. What's a simpler, more general approach? What would the Macintosh team in 1984 have done?

Well, we don't know, but thanks to time travel we can look ahead into the future of 1991, where System 7 introduced three things:  (1) the Apple Menu and (2) a Launcher that were both populated by (3) Aliases. Instead of a dedicated editor for menu entries (e.g. Font/DA Mover), you just combine the primitives - file selection, alias creation, and file drag/drop. DeskTop _already_ did this for the DESK.ACC folder (now called APPLE.MENU). What's missing are Aliases.

So... I'm thinking of introducing a Shortcut file type. (It took ~15 minutes to code up support, but I'm not sure I'll incorporate it any time soon.)

The idea is to replace the entire Shortcuts-as-a-special-file-and-dedicated-editor with a menu populated from a folder (maybe SHORTCUTS/), and a "Make Shortcut" command that makes a shortcut to the selected icon, and drops it in that folder. No more need for Edit/Delete/Run a Shortcut commands - they're just files in a folder.  The optional Selector module just enumerates the directory, shows the first 24 items. Easy peasy!

The down sides include:
* can't put special characters in the names (because they're just filenames)
* can't explicitly distinguish entries in the menu vs. just in the dialog (but does anyone really understand that?)
* refreshing the menu dynamically when the folder is modified is tricky (but worth it? could apply to Apple Menu too #301)
* no way to edit a shortcut, or diagnose a problem with it. Do we need "Show Original" ?

This also lets us extend the number of items in the Shortcuts menu (#299) to 11 (14 total items minus "Make Shortcut", "Show Shortcuts", and a separator)

**Comment by inexorabletash** [comment]
Oh, another big downside is controlling "copy to RAMCard" on startup/run. Gotta figure that out too.

**Comment by xandark** [comment]
Yes, this feels to me more elegant than what the MouseDesk developers crafted. 

Maybe the "copy to RAMcard" functionality could just be a separate sub-folder inside SHORTCUTS, like SHORTCUTS/PRELOAD ?  Or if that's too wonky, maybe changing the Auxtype of the shortcut file?

**Comment by xandark** [comment]
It looks like the Make Alias feature is accomplishing this. I like the LNK files.
